src/main/index.js

```diff
@@ -167,13 +167,26 @@ init_electron_store();
 
 // start the Flask server
 boot_up_flask();
+
+// save customer id after it's verified in the /get_auth aws route
+ipcMain.on("save_customer_id", (e, customer_account) => {
+  e.reply("save_customer_id", 200);
+
+  const { cust_id, pass_key } = customer_account;
+  store.set("customer_account_id", {
+    id: cust_id,
+    password: pass_key,
+  });
+});
+
```

Comment:
sets customer account to electron store to be set in .yaml file after validated in aws  /get_auth route